src/database/DatabaseInteractions.php

```diff
@@ -279,7 +293,7 @@ function canMakeRequestForUsername(string $username, IDatabase $dbr) : bool {
 }
 
 function getBlocks(IDatabase $dbr) : array {
-	$result = $dbr->select('scratch_accountrequest_block', ['block_username', 'block_reason'], [], __METHOD__, ['ORDER BY' => 'block_timestamp ASC']);
+	$result = $dbr->select('scratch_accountrequest_block', ['block_username', 'block_reason', 'block_expiration_timestamp'], [], __METHOD__, ['ORDER BY' => 'block_timestamp ASC']);
```

Comment:
One thing here: there should also be a WHERE condition to not get expired blocks